### ISSUE TITLE
An error notification came up after publishing event

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#501]
**Describe the bug**
I made a new event

The result was this:

![Screenshot 2022-12-18 at 20 10 43](https://user-images.githubusercontent.com/5323259/208330184-efa2361f-5140-47d8-97ce-7ae3c1eb5efe.jpg)

"Event added. It will now show on the calendar. No link was generated for this event. There may be an unknown error."

I tested the Calendar and they mostly seem to be working alright.
Associated issue https://github.com/joedolson/my-calendar/issues/498

**Comment by joedolson** [comment]
If you can run Query Monitor and check for a DB error from that, it would be great; but this is a response that's checking against an unknown error, as it says, so I have basically nothing to go on here.

**Comment by paaljoachim** [comment]
I checked Settings - General. Drop MySQL tables on uninstall and delete plugin settings on uninstall. Then deleted the plugin and downloaded it from Github again.
I installed and activated Query Monitor.

Made a new event and filled in the various fields.
Again I noticed the same unknown error.

![Screenshot 2022-12-18 at 23 13 34](https://user-images.githubusercontent.com/5323259/208346296-6518f5d0-0ea9-480f-afee-5c9fc8198d28.jpg)

--

![Screenshot 2022-12-18 at 23 14 11](https://user-images.githubusercontent.com/5323259/208346315-41d63ed1-35da-453d-ba2e-292dfa93d19c.jpg)


I am not sure what is most important to look closer at.
